test/bagofwords_transformer.jl

```diff
@@ -0,0 +1,78 @@
+using MLJBase
+using TextAnalysis
+
```

Comment:
Just a suggestion - feel free to ignore. It seems the two `AbstractTextTransformers` share some behaviour that could be tested simultaneously to avoid code duplication and make maintenance easier. Maybe you could wrap those tests separately, as in 

```julia
for model in [:TfidfTransformer, :B25Transformer]
   quote
        @test some_property($model)
        ...
    end |> eval
end
```